test/testing.js

```diff
@@ -0,0 +1,14 @@
+var nuts = require('../lib');
+
+var config = {
+    repository: 'SamyPesse/nuts-testing',
```

Comment:
Wouldn't it be more handy to implement and use a `TestBackend` (inherits from `Backend`) in which we could inject mock data? It would permit to test a large range of scenarios.

The `GithubBackend` looks thin and dumb enough so that test `TestBackend` will cover a similar amount of code. Moreover, some test cases (functional) can use the `GithubBackend` with the `nuts-testing` repo.